src/main/kotlin/com/cognifide/gradle/aem/tooling/tail/TailConfig.kt

```diff
@@ -1,12 +1,20 @@
 package com.cognifide.gradle.aem.tooling.tail
 
+import com.cognifide.gradle.aem.tooling.tasks.Tail
 import org.gradle.api.tasks.Input
 
 class TailConfig {
 
     @Input
     val filters = mutableListOf<(Log) -> Boolean>()
 
+    @Input
+    val blacklistFiles = mutableListOf<String>(Tail.DEFAULT_BLACKLIST_FILE)
```

Comment:
I prefer naming `[TOPIC]_[VARIANT]` because then similar topic is grouped by same prefix (values are not spreading)

NAME_DEFAULT vs DEFAULT_NAME

SCHEDULER_EXPR_DAILY
SCHEDULER_EXPR_DEFAULT
etc

Reply:
OK